components/cldr-json-data-provider/src/error.rs

```diff
@@ -1,12 +1,16 @@
 use std::error;
 use std::fmt;
 
+#[cfg(feature = "download")]
+use crate::download::error::Error as DownloadError;
+
```

Comment:
since `download` is optional, would it be easier to maintain to have the `From` in `download::error` instead of here?

Reply:
Perhaps, but I still need to put the Display and std::error::Error implementations in this file, so it seems odd that From would be in a separate place.  It's easy to put it in this file, and I protect it with `#[cfg(feature = "download")]`.